go/scrape/scrape.go

```diff
@@ -3,43 +3,84 @@ package scrape
 import (
 	"errors"
 	"strconv"
-)
+	"strings"
 
-type KyukoData struct {
-	Place      int
-	Week       int
-	Period     int
-	Date       string
-	ClassName  string
-	Instructor string
-	Reason     string
-}
+	"github.com/PuerkitoBio/goquery"
+	iconv "github.com/djimenez/iconv-go"
```

Comment:
こたえ

```
Also, because the net/html parser requires UTF-8 encoding, so does goquery: it is the caller's responsibility to ensure that the source document provides UTF-8 encoded HTML. See the wiki for various options to do this.
```

```
8a 77 学
89 ef 会
c2 a0 NBSP (UTF-8)
0a 09 09 09 09 09 09 09 09 WTF!?
```